README.md

```diff
@@ -189,6 +189,13 @@ This only applies when a custom [`body`](#body) is not passed in. Defaults to:
 ]
 ```
 
+#### `retryMethods`
+
+On which methods should the connection be retried in case of socket hang up.  
+**Be aware** that setting here not idempotent method may lead to unexpected results on target.
+
+By default: `GET`, `HEAD`, `OPTIONS`, `TRACE` 
```

Comment:
Can't seem to put this inside a suggestion. 😞 
I would suggest changing this to something like the following, to indicate it accepts an array of options, as at present it the documentation makes it looks like it accepts a single string:

Defaults to:
```js
[ 'GET', 'HEAD',  'OPTIONS', 'TRACE' ]
```

Reply:
Done